pkg/util/ipvs/kernelcheck_linux.go

```diff
@@ -75,18 +75,21 @@ func (r RequiredIPVSKernelModulesAvailableCheck) Check() (warnings, errors []err
 			return nil, errors
 		}
 
-		builtInModules := sets.NewString()
-		for _, builtInMode := range ipvsModules {
-			match, _ := regexp.Match(builtInMode+".ko", out)
+		var missingModules []string
+		for _, module := range modules {
+			match, _ := regexp.Match(module+".ko", out)
 			if !match {
-				builtInModules.Insert(string(builtInMode))
+				out, err = r.Executor.Command("modinfo", module).CombinedOutput()
+				if err != nil {
+					warnings = append(warnings, fmt.Errorf("error getting %s module info: %v: %s", module, err, strings.TrimSpace(string(out))))
```

Comment:
`error getting %s module info: %v: %s`
->
`error getting module info for %q; error: %v; output: %s`

and a related question, do we benefit from CombinedOutput() here?
can you please show example output?

if not we can omit `; output: %s`

Reply:
I think we do:
```
# modinfo bla
modinfo: ERROR: Module bla not found.
$ modinfo bla
-bash: modinfo: command not found
```